Makefile

```diff
@@ -138,6 +138,23 @@ docker-build: test ## Build docker image with the manager.
 docker-push: ## Push docker image with the manager.
 	docker push ${IMG}
 
+# PLATFORMS defines the target platforms for  the manager image be build to provide support to multiple
```

Comment:
was this mandatory for the upgrade?

Reply:
Yeah it was part of the operator-sdk upgrade